tests/unit/test_openapi/test_parameters.py

```diff
@@ -1,4 +1,5 @@
 from typing import TYPE_CHECKING, List, Optional, Type, cast
+from uuid import UUID
```

Comment:
I could import this under `TYPE_CHECKING` but then I would have to import it in runtime (inside the test) and add it using `signature_types`. So I imported it here, lmk if this has to be changed